scstadmin/scstadmin.spec.in

```diff
@@ -16,7 +16,7 @@ URL:		http://scst.sourceforge.net/
 Source:		%{name}-%{version}.tar.bz2
 BuildRoot:	%{_tmppath}/%{name}-%{version}-build
 AutoReqProv:	yes
-Requires:	%{expand:%%(for c in killall rm; do echo -n "$(which $c) "; done)}
```

Comment:
It is on purpose that paths of binaries appear here instead of package names. These paths come from the distro on which the RPM is built while there is no guarantee that all RPM-based distros use the psmisc and coreutils RPM names.

Reply:
Hi Bart,

Thank you for the comment!

> These paths come from the distro on which the RPM is built while there is no guarantee that all RPM-based distros use the psmisc and coreutils RPM names.

However, what will happen if the distro doesn't have killall? The which command will fail to find the path, resulting in a missing dependency in the Require line. The Require line is specifically designed for cases when a dependency is missing, so the current solution doesn't help to resolve this problem.

This also may lead to issues like the one seen here: https://github.com/SCST-project/scst/issues/152

I understand that there is no guarantee that all RPM-based distros use psmisc, but at least it will work better than the current solution. Unfortunately, I don't know exactly how to solve this problem in a better way.

Thanks,
Gleb